tests/state/test_state_api.py

```diff
@@ -180,3 +181,24 @@ def do_job():
             {"name": "do_work", "priority": 0, "queue_name": "default"},
             {"name": "do_job", "priority": 10, "queue_name": "foo"},
         ]
+
+    def test_filter_messages_id(self, stub_broker, api_client, state_middleware):
+        random_states = []
+        for i in range(10):
+            random_state = State(
+                "id{}".format(i),
+                choice(list(StateNamesEnum)),
+                args=sample(range(1, 10), randint(0, 5)),
```

Comment:
you do not need to set args and kwargs, as their are useless for the test. They bring noise making it more difficult to uderstand.